src/routes/console/project-[project]/databases/database-[database]/+layout.svelte

```diff
@@ -1,5 +1,28 @@
+<script lang="ts">
+    import { goto, invalidate } from '$app/navigation';
+    import type { Models } from '@aw-labs/appwrite-console';
+    import Create from './create.svelte';
+    import { showCreate } from './store';
+    import { base } from '$app/paths';
+    import { page } from '$app/stores';
+    import { Dependencies } from '$lib/constants';
+
+    const project = $page.params.project;
+    const databaseId = $page.params.database;
+
+    async function handleCreate(event: CustomEvent<Models.Collection>) {
+        $showCreate = false;
+        await invalidate(Dependencies.DATABASE);
+        await goto(
+            `${base}/console/project-${project}/databases/database-${databaseId}/collection-${event.detail.$id}`
+        );
+    }
+</script>
+
 <svelte:head>
     <title>Database - Appwrite</title>
 </svelte:head>
 
 <slot />
+
+<Create bind:showCreate={$showCreate} on:created={handleCreate} />
```

Comment:
This is for creating a collection right?

If its now used across layouts, lets give it a clearer name. `showCreateCollection`